tests/performance/collection_feed/footype.sd

```diff
@@ -0,0 +1,41 @@
+search footype {
+document footype {
+
+  field arr_attr_long_nofs type array<long> {
```

Comment:
Are we planning to use the array type fields? If not, consider removing them.

Reply:
I was thinking of adding a separate test case for arrays since they have also been touched by the backend work, but I can remove them until that is actually present.